index.js

```diff
@@ -79,9 +73,8 @@ module.exports = {
 
 
                     var imageDir = path.dirname(path.resolve(screenShotPath, imageName));
-                    if (!fs.existsSync(imageDir)) {
```

Comment:
The if is not necessary. On an existing directory, mkdirp returns null (without exceptions). 
Additionally mkdir -p works fine with existing directories too inside of the terminal, so there is no need for an explicit hint.